week2/2-website/css/style.css

```diff
@@ -15,5 +42,199 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+img {
+  max-width: 100%;
+}
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  border-bottom: 1px solid #ddd;
+  width: 100%;
+}
+.logo {
+  width: 2.5rem;
+  margin-left: 15rem;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-around;
+  margin-bottom: 0.5rem;
+  margin-right: 5rem;
+}
+
+.navbar ul {
+  display: flex;
+  list-style: none;
+}
+
+.navbar li {
+  padding: 1rem;
+  color: #838994;
+  font-size: 1.7rem;
+  letter-spacing: 0.1rem;
+  text-decoration: none;
+}
+.menu li a:hover {
+ color: #333333;
+}
+li a.active {
+  color: #4c5058;
+  font-weight: 500;
+}
+.card img {
+  width: 12.7rem;
+  margin-bottom: 2.5rem;
+  margin: 2.5rem;
+  padding: 2.5rem;
+}
+.about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
+.about .container {
+  display: flex;
+  align-content: space-around;
+  align-items: center;
+  margin-bottom: 10rem;
+}
+.about h2 {
+  margin-top: 7rem;
+  font-size: 5rem;
+  font-weight: 300;
+}
+.card {
+  font-size: 2.5rem;
+}
+.card h3 {
+  font-size: 2.5rem;
+  font-weight: 300;
+  text-align: center;
+}
+.hero {
+  background-size: cover;
+  background: linear-gradient(to top, rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.5)),
+    url(../img/first-background.jpg) center no-repeat;
+  margin-bottom: 3rem;
+  color: #ddd;
+  height: 65rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.hero-heading {
+  font-size: 5rem;
+  font-weight: 300;
+  margin-bottom: 0;
+}
+.hero-paragraph {
+  font-size: 2.5rem;
+  font-weight: 300;
+  letter-spacing: 0.2rem;
+  margin-top: 0;
+  margin-bottom: 2rem;
+}
+.hero-button {
+  width: 15rem;
+  padding: 1.2rem 2rem;
+  letter-spacing: 0.1rem;
+  border: none;
+  border-radius: 0.5rem;
+  outline: none;
+  cursor: pointer;
+  background-color: #f15a29;
+  border-color: #f15a29;
+  text-shadow: none;
+  color: white;
+  font-size: 1.5rem;
+}
+.hero-button:hover {
+  background-color: #d9400e;
+  border-color: #cf3d0e;
+  cursor: pointer;
+}
 
+hr {
+  border: 0;
+  border-top: 1px solid #eaebec;
+  max-width: 65%;
+  margin: auto;
+}
+
+.social {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+.social h3 {
+  font-size: 1.5rem;
+  font-weight: 300;
+  text-align: center;
+}
+.social ul {
+  list-style: none;
+  display: flex;
+  text-align: center;
+}
+.social li {
+  width: 4rem;
+  display: inline-block;
+  margin: 1.25rem;
+  padding: 1.5rem 0;
+  text-align: center;
+}
+i.fa {
+  display: inline-block;
+  border-radius: 6rem;
```

Comment:
I think you want to be adding styling to the the `li` item rather than the icon itself.

This will ensure that the different widths of the icons doesn't affect the layout.